### ISSUE TITLE
Task Blocks

### ISSUE THREAD

**Issue by nilaeus** [loop-revolution/blocks#658]
Things on your to-do list
- [x] Metadata (name, about)
- [x] Stage (not done, done, cancelled)
- [x] Dependencies (blocked by another task)
- [x] Due dates (due before this time)
- [x] Date to start (Date to do a task on)
- [ ] Repeat interval (in how much time does this repeat)
